portal-api/src/v1/routes.js

```diff
@@ -251,7 +251,18 @@ authRouter
     bankIdValidation,
     handleValidationResult,
     validateUserAndBankIdMatch,
-    getDueReportDates,
+    utilisationReportControllers.getDueReportDates,
+  );
+
+authRouter
+  .route('/banks/:bankId/utilisation-report-download/:_id')
+  .get(
+    validateUserHasAtLeastOneAllowedRole({ allowedRoles: [PAYMENT_REPORT_OFFICER] }),
+    bankIdValidation,
+    handleValidationResult,
+    validateUserAndBankIdMatch,
+    validateMongoId,
```

Comment:
we're calling this after the handleValidationResult so we won't handle any validation errors from this here